### PR TITLE
Suppress logging in tests

### LINKED ISSUE #768: Better logging management
Having `ProtocolParameters` delivered to your terminal on each `Contract` run is annoying.

We should find a way to collect Contract logs and show them only in case the test fails

Blocked by https://github.com/Plutonomicon/cardano-transaction-lib/issues/730

### PR DESCRIPTION
**Pull request by klntsky**
Closes #768

### Pre-review checklist

- [x] All code has been formatted using our config (`make format`)
- [x] Any new API feature includes a full example or integration test run with Plutip on CI
- [x] The changelog has been updated under the `## Unreleased` header, using the appropriate sub-headings (`### Added`, `### Removed`, `### Fixed`), and the links to the appropriate issues/PRs have been included

### CODE REVIEW

File: templates/ctl-scaffold/test/Main.purs [hunk 1]
```diff
@@ -52,4 +52,5 @@ config =
       , dbname: "ctxlib"
       }
   , customLogger: Nothing
+  , suppressLogs: false
```
Comment (ngua) [inline templates/ctl-scaffold/test/Main.purs:55]: Please update the CTL revision in the template's flake and `packages.dhall`

File: src/Contract/Monad.purs [hunk 2]
```diff
@@ -204,6 +208,7 @@ type ConfigParams (r :: Row Type) =
   , logLevel :: LogLevel
   , walletSpec :: Maybe WalletSpec
   , customLogger :: Maybe (Message -> Aff Unit)
+  , suppressLogs :: Boolean
```
Comment (ngua) [inline src/Contract/Monad.purs:211]: I think this field could use a doc comment

**Review by ngua — CHANGES_REQUESTED** [review]

File: templates/ctl-scaffold/test/Main.purs (hunk 1 above)
Comment (klntsky) [reply templates/ctl-scaffold/test/Main.purs:55]: We can't update the revision until we know it (that is after we merge the PR)

Comment (ngua) [reply templates/ctl-scaffold/test/Main.purs:55]: Not necessarily, you can set it to head on the branch you're working on

**Review by ngua — APPROVED** [review]

**Pull request merged**